### PR TITLE
Add slider to change transparency

### PR DESCRIPTION
**Pull request by yycking**
- redraw knob
- use knob's image as static file

### CODE REVIEW

File: Touch Bar Simulator/ToolbarSlider.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//  Created by Wayne Yeh on 2017/3/20.
+//  Copyright © 2017年 Sindre Sorhus. All rights reserved.
+//
+
+import Cocoa
+
+private class ToolbarSliderCell: NSSliderCell {
+	
+	class var knob: NSImage {
+		let frame = NSRect(x: 0, y: 0, width: 32, height: 32)
```
Comment (sindresorhus) [inline Touch Bar Simulator/ToolbarSlider.swift]: Add a `print("draw")` inside here and you'll see that it's not just drawn once, but many times when dragging.

Comment (yycking) [inline Touch Bar Simulator/ToolbarSlider.swift]: You are right.
`knob` is not static object.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by yycking** [comment]
OK!
I had merge then.

**Comment by sindresorhus** [comment]
This is excellent. Thank you @yycking :)

**Pull request merged**